asistente_ladm_col/logic/quality_rules/qr_overlapping_right_of_ways_buildings.py

```diff
@@ -0,0 +1,118 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-06-20
+        git sha         : :%H$
+        copyright       : (C) 2022 by Sergio Ramírez (SwissTierras Colombia)
+        email           : sramirez@colsolutions.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+import json
```

Comment:
Unused import

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/8c3cdd521b8b648d49ea12f26f170f6f6424f6dd